examples/nextjs-scheduler/app/utils/handlePeers.ts

```diff
@@ -0,0 +1,41 @@
+import { Indexable } from 'yorkie-js-sdk';
+
+const randomPeers = [
+  'John',
+  'Alice',
+  'Steven',
+  'Kate',
+  'Daniel',
+  'Chang',
+  'Marie',
+  'Fred',
+  'Sanchez',
+  'Kim',
+  'Wayne',
+  'Seon',
+  'Diaz',
+  'Tom',
+];
```

Comment:
How about replacing peer names with characters from [alice and bob](https://en.wikipedia.org/wiki/Alice_and_Bob)?

Reply:
@hackerwins work done! 😃